tests/test_dvc.py

```diff
@@ -204,10 +176,12 @@ def test_no_scm_repo(tmp_dir, mocker):
     assert live._save_dvc_exp is False
 
 
-def test_dvc_repro(tmp_dir, monkeypatch, mocker):
+def test_dvc_repro(tmp_dir, monkeypatch, mocked_dvc_repo, mocked_studio_post):
     monkeypatch.setenv(DVC_ROOT, "root")
-    mocker.patch("dvclive.live.get_dvc_repo", return_value=None)
     live = Live(save_dvc_exp=True)
+    assert live._baseline_rev is not None
+    assert live._exp_name is not None
+    assert not live._studio_events_to_skip
```

Comment:
Consolidated from the deleted `test_exp_save_run_on_dvc_repro` above. On `repro`, everything will still work except actually saving an exp ref.